File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageHistoryResult.java
package com.dhsdevelopments.potato.clientapi.message;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MessageHistoryResult
{
    @SerializedName( "messages" )
    private List<Message> messages;

    public List<Message> getMessages() {
        return messages;
    }

    @Override
    public String toString() {
        return "MessageHistoryResult[" +
                       "messages=" + messages +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageHistoryResult.kt
package com.dhsdevelopments.potato.clientapi.message

import com.google.gson.annotations.SerializedName

class MessageHistoryResult {
    @SerializedName("messages")
    lateinit var messages: List<Message>

    override fun toString(): String {
        return "MessageHistoryResult[messages=$messages]"
    }
}
